network/server/tchannelthrift/node/service.go

```diff
@@ -611,6 +611,24 @@ func (s *service) SetPersistRateLimit(
 	return s.GetPersistRateLimit(ctx)
 }
 
+func (s *service) GetWriteNewSeriesAsyncResult(
+	ctx thrift.Context,
+) (*rpc.NodeWriteNewSeriesAsyncResult_, error) {
+	value := s.db.RuntimeOptionsManager().Get().WriteNewSeriesAsync()
```

Comment:
Perhaps we should do the same for rate limit values to be consistent? I'm fine with doing it in a separate diff also.